### PR TITLE
Add Sendbird w/ npm auto-update

### PR DESCRIPTION
**Pull request by tylerhammer**

### CODE REVIEW

File: packages/s/Sendbird.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (MattIPv4) [inline packages/s/Sendbird.json:1]:
```suggestion
```
Remove newline from start for consistency.

File: packages/s/Sendbird.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+  
+{
+  "name": "sendbird",
```
Comment (MattIPv4) [inline packages/s/Sendbird.json:3]:
```suggestion
  "name": "Sendbird",
```
`name` should match the filename. Either `name` needs to become `Sendbird` or the fle should be renamed to `sendbird.json`.

File: packages/s/Sendbird.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "version": "",
```
Comment (MattIPv4) [inline packages/s/Sendbird.json:4]:
```suggestion
```
`version` isn't needed, the attr can be removed.

File: packages/s/Sendbird.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "npmName": "sendbird",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "SendBird.min*"
+      ]
+    }
+  ]
```
Comment (MattIPv4) [inline packages/s/Sendbird.json:26]:
```suggestion
  "autoupdate": {
    "source": "npm",
    "target": "sendbird",
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "SendBird.min*"
        ]
      }
    }
  ]
```
Please use the new `autoupdate` format documented in CONTRIBUTING.md

**Comment by tylerhammer** [comment]
Thanks for the comments @MattIPv4. I went ahead and submitted with your suggested changes.

File: packages/s/sendbird.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    "url": "https://github.com/sendbird/SendBird-SDK-JavaScript"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "sendbird",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "SendBird.min*"
```
Comment (MattIPv4) [inline packages/s/sendbird.json:23]:
```suggestion
          "SendBird.@(d.ts|min.js)"
```
cdnjs supports hosting ts files, so we might as well include the typings for anyone using this via libman in an IDE etc.

**Comment by tylerhammer** [comment]
Good catch!

**Pull request merged**